src/Fantomas.Tests/NumberOfItemsTests.fs

```diff
@@ -0,0 +1,303 @@
+module Fantomas.Tests.NumberOfItems
```

Comment:
Another nitpick I'm afraid but the module name should end on the word `Tests` to match the file name.

Reply:
Sorry about that, changed.